app/src/main/java/com/github/onlynotesswent/ui/overview/AddNote.kt

```diff
@@ -72,7 +73,7 @@ fun AddNoteScreen(
   var className by remember { mutableStateOf("") }
   var classCode by remember { mutableStateOf("") }
   var classYear by remember { mutableIntStateOf(currentYear) }
-  var template by remember { mutableStateOf("Choose An Option") }
+  var template by remember { mutableStateOf("Choose an option") }
```

Comment:
Here I am not sure if we should stick to lower or capital letters at the start of each word. Maybe we should discuss this in a meeting with all team members